src/numpy.jl

```diff
@@ -64,7 +64,12 @@ function npyinitialize()
     numpy = pyimport("numpy")
 
     # emit a warning if both Julia and NumPy are linked with MKL (#433)
-    if :mkl in LinearAlgebra.BLAS.get_config().build_flags &&
+    if isless(VERSION,v"1.1")
+        julia_compiled_with_mkl = occursin("MKL", LinearAlgebra.BLAS.openblas_get_config() )
```

Comment:
Why not just use the old code `LinearAlgebra.BLAS.vendor() === :mkl` in versions prior to `1.7`?

Reply:
If that is the right way to go about it then fine; I don't have enough seniority to see results of checks to know what works in what version. What I proposed seems to work in 1.0.5 so that's what I went with. Regarding the `isless` I received errors when trying to compare to `VersionNumber` so tried to figure an alternative. I see that a simple < works with `VERSION`. It's all a learning process.